discord/user.py

```diff
@@ -383,6 +391,24 @@ def blocked(self):
 
         http = self._state.http
 
+        try:
+            house = fields['house']
+            if house is None:
+                await http.leave_hypesquad_house()
+
+            elif 'first_name' not in fields or 'last_name' not in fields:
```

Comment:
Generally speaking I prefer accessing and returning than checking if they key exists. I think this probably applies here too since it's two fields.

Reply:
I'm actually not too sure what you mean by this @Rapptz